Week 01/assignment/index.css

```
@@ -0,0 +1,202 @@
+@import './reset.css';
+
+body {
+  background-color: var(--backGround);
+  color: var(--mainColor);
+}
+
+.home {
+  position: relative;
+  height: 70vh;
+  width: 100vw;
+}
+
+/* 헤더 */
+.home > header {
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  z-index: 1;
+  width: 100%;
+  height: 15%;
+  padding: 0 5%;
+  background: none;
+}
+
+.header__right,
+.header__left {
+  display: flex;
+  font-size: 1.5rem;
+}
+
+.header__left {
+  width: 55%;
+}
+
+.header nav ul {
+  display: flex;
+  justify-content: space-between;
+  width: 40%;
+}
+
+.header > nav li:hover {
+  border-bottom: dotted 2px var(--blurBackGround);
+}
+
+.header__right {
+  justify-content: end;
+  width: 20%;
+}
+
+/* 홈 배너 */
+.home_banner__img {
+  width: 100%;
+  height: 100%;
+}
+
+.header__title,
+.banner_list--title,
+.scroll_view--title {
+  font-size: 2rem;
+  font-weight: 900;
+  margin-right: 3rem;
+  font-family: sans-serif;
+}
+
+.home_banner__etc {
+  top: 70%;
+  left: 70%;
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.home_banner__etc > span > p {
+  font-size: 3rem;
+  color: var(--mainColor2);
+  font-weight: 800;
+}
+
+.home_banner__button {
+  display: flex;
+  justify-content: center;
+  gap: 1rem;
+  padding: 10px;
+  border-radius: 10px;
+  background-color: var(--blurBackGround);
+  width: 40%;
+  margin-top: 1rem;
+}
+
+.home_banner__button i,
+.home_banner__button p {
+  font-size: 1rem;
+}
+
+.home_banner__button:hover {
+  background-color: var(--buttonHover);
+}
+
+.home_banner__button:hover i,
+.home_banner__button:hover p {
+  color: var(--textHover);
+}
+
+/* 무한배너 */
+.infinite_banner,
+.scroll_view {
+  margin: 5rem 2rem 0 2rem;
+}
+.infinite_banner__photos {
+  display: flex;
+  width: 100%;
+  margin-top: 3rem;
+  overflow: hidden;
+}
+
+.banner__list {
+  animation: scroll 18s linear infinite;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.banner__list > li {
+  margin-left: 2rem;
+  transition: transform 0.3s ease;
+}
+
+.banner__list > li:hover {
+  transform: scale(1.13);
+}
+
+.banner__list > li > img {
+  width: 300px;
+  height: 300px;
+  transition: transform 0.3s ease;
+}
+
+.l1:last-child {
+  animation-delay: 20s;
+}
+
+@keyframes scroll {
+  0% {
+    transform: translateX(0);
+  }
+  100% {
+    transform: translateX(-100%);
+  }
+}
+
+/* 스크롤뷰 */
+.scroll_view__photos {
+  display: flex;
+}
+
+.scroll_view__photos a {
+  width: 3%;
+  margin-top: 3rem;
+  align-content: center;
+  text-align: center;
+  background-color: var(--button);
+  border: none;
+  border-radius: 0.2rem;
+  cursor: pointer;
+}
+
+.scroll_view__photos > ul {
+  margin-top: 3rem;
+  display: flex;
+  overflow-x: scroll;
+}
```

```suggestion
  scroll-behavior: smooth;
}
```
앗 이미 아실수도 있겠지만, 요거 한줄 추가해주면 스크롤 넘어갈때 부드럽게 넘어가게 해줄 수도 있습니다!

---

넵넵! 반영해보도록 하겠습니다 :)